deployment/lambdas/submit_job/lambda_entrypoint.py

```diff
@@ -89,6 +93,27 @@ def main(event, context):
     data = get_submission_data(tumor_sample_md, normal_sample_md, subject_id, api_auth)
     LOGGER.debug(f'compiled submission data: {data}')
 
+    # Abort job submission, if job_name is in gpl-job-queue
```

Comment:
That's to prevent the same job being submitted multiple time? 
This checks against the SQS queue to see if a same name job already exists?

Reply:
Correct. Only limited to the same batch window (batch queue will be cleared away) but this is expected. 

Just to prevent - User might click couple times, consecutively. Though UI will try best to prevent it but, still possible due to XHR async call nature.

Reply:
Would it make sense to add/increase a delay on the queue itself so new jobs are available longer (for this check) in the queue before being cleaned/send off?

Reply:
Yup, certainly. Plan is to trial & test in DEV, tuning this. If thing get complex then probably might have to offload "a state" in DynamoDB or so. But, MVP first, yes. :)

Reply:
Awesome!